AppointEase/AppointEase.Application.Contracts/Models/DatabaseModels/TblPacient.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace AppointEase.Application.Contracts.Models.DbModels;
+
+public partial class TblPacient
+{
+    [Key]
+    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+    public int UserId { get; set; }
+
+    public string PersonalNumber { get; set; } = null!;
+
+    public string Name { get; set; } = null!;
+
+    public string Surname { get; set; } = null!;
+
+    public string Gander { get; set; } = null!;
```

Comment:
typo.........

Reply:
Also why are you giving the default null for string properties? The value of a default string is always null?

Reply:
No, the reason it's invalid is because of the validation that if the client doesn't fill anything in the input section, then the message appears that the field must be completed.

Reply:
That should be a front-end validation?

Reply:
We should always check if we have the correct data before even CALLING database methods, so we have front-end validation going on, we will have even validation at our own layers so when you call the repository methods it's all been corrected